content/posts/fleek-network-managing-the-key-store.md

```diff
@@ -0,0 +1,193 @@
+
+[server_config]
+port = 4069
+addr = "0.0.0.0"
+```
+
+⚠️ Beware that the configuration file might look a bit different depending on the version you're running and the current development features in place by the Fleek Network dev team.
+
+Some advanced use-cases might require to deal with the host and port number binding, depending on the service provider needs and customisation. We'll keep things simple and assume the default settings!
+
+In this guide, we are interested in the `network_config` section settings for the `keystore_path` which defaults to `/root/.ursa/keystore`.
+
+💡 The identity is a text description for the keystore and at time of writing we have not yet implemented multiple identity management via the CLI. Thereupon, the identity value serves to find the filename match in the `keystore_path`. This might change as we progress with development. Check the [Identity selection](#identity-selection) to learn more about it!
+
+### Identity selection
+
+Multiple identity management is yet to be implemented but in any case, some users might find it trivial to keep multiple identities and switch between them referencing them by name.
+
+Let's suppose that we've recently moved to a new server setup and copied our previous server keystore as `old-server-keystore.pem` and renamed the `$HOME/.ursa/keystore/default.pem` to `new-server-keystore.pem`.
+
+Here's an example of how our `$HOME/.ursa/keystore` directory would look like:
+
+```sh
+.
+├── new-server-keystore.pem
+└── old-server-keystore.pem
+
+0 directories, 2 file
+```
+
+The `old-server-keystore.pem` is the original keystore and `new-server-keystore.pem` a brand new identity we got when completed setup the node in the new system.
+
+As we have our `network_config` -> `identity` set as `default` and `network_config` -> `keystore_path` defined as `/root/.ursa/keystore`, we have to change it! We can switch to an existing identity by providing the filename before the node initialisation! 
+
+Set the identity name in the `network_config` -> `identity` to a matching PEM file that exists in the `keystore_path`. For our example, that'd be `new-server-keystore.pem` or `old-server-keystore.pem`.
+
+💡 Name the PEM files as you wish, we have used the naming above to make it as clear as possible.
+
+## Identity
+
+When you run a node, there's a private key which is used to compute a public key that your node is identified with. 
+
+If another party holds the private key, they can control the identity, e.g. have access to the rewards. For this reason, it's important to have at the very least a basic understanding of how identity works in Fleek Network.
+
+### Type of keys
+
+When you run a node, there's a private key which is used to compute a public key that your node is identified with. If another party holds the private key, they can get get control of your rewards. It's important to have a very basic understanding of how identity works in Fleek Network.
```

Comment:
Should we omit this part since we already said it above?

Reply:
Good catch, that was supposed to not be there :)